plugins/splunk-on-call/src/components/EntitySplunkOnCallCard.tsx

```diff
@@ -92,7 +102,9 @@ export const EntitySplunkOnCallCard = () => {
       },
       {},
     );
-    return { usersHashMap, userList: allUsers };
```

Comment:
I wasn't able to find any reference to `userList` and it appears only `usersHashMap` is being used, so I removed this.